examples/dialog-modal/js/dialog.js

```diff
@@ -127,8 +127,11 @@ aria.Utils = aria.Utils || {};
   aria.Dialog = function (dialogId, focusAfterClosed, focusFirst) {
     this.dialogNode = document.getElementById(dialogId);
     var role = this.dialogNode.getAttribute('role');
+    var isDialog = role.split(' ').some(function (r) {
```

Comment:
I'd suggest `trim()`ing `r` as well.

Does this code need to check for `this.dialogNode === null`? Wouldn't the `getAttribute('role')` call at line 129 already fail?

Reply:
Good call! I added a fix.